### DISCUSSION TITLE
Finding update/upgrade related log from ipamd

### DISCUSSION THREAD

**Discussion by cinmoy-vivasoft** [aws/amazon-vpc-cni-k8s#2462] [Q&A]
I suddenly found out in my eks console that the vpc-cni addon was showing "Update failed" when asked that no one in my developer team was not able to watch logs of the pods. As far as I know that vpc-cni does not auto update.

At the time of occurance -

-  eks version 1.26
- vpc-cni version v1.12.6-eksbuild.1

Logging is disabled for my cluster and events got deleted.
So my question is is there any way to find logs about failed update (directly or indirectly) from the ipamd.log or plugin.log  in my node?

**Comment by jdn5126** [comment]
The VPC CNI does not auto-update, so it sounds like you are referring to an error in the AWS console using EKS Managed Addons. Please file an issue at https://github.com/aws/amazon-vpc-cni-k8s/issues for cases like this in the future. Closing this as an outdated issue.
